node/pkg/telemetry/telemetry.go

```diff
@@ -14,16 +16,39 @@ import (
 	"google.golang.org/api/option"
 )
 
+const TELEMETRY_LOG_LEVEL = zap.InfoLevel
```

Comment:
small nit, Constants in go are not capital snake case but camel case

Reply:
changed